test/functionalTest/cases/0000_statistics_operation/statistics_with_semaphores.test

```diff
@@ -38,80 +38,49 @@ echo
 echo "+++++ 2. statistics in JSON"
 orionCurl --url /statistics --json
 echo
-
-
-#
-# About
-#   <versionRequests>REGEX((0|1))</versionRequests>   AND
-#   "versionRequests": "REGEX((0|1))"
-# When running under valgrind, an additional "GET /version" is issued to make sure the broker is running
-#
-#
-# About
-#   "subCacheRefreshs": "REGEX((0|1))"   AND
-#   <subCacheRefreshs>REGEX((0|1))</subCacheRefreshs>
-# When the broker is started with -noCache, there are no refreshes performed
-#
+echo
 
 --REGEXPECT--
 +++++ 1. statistics in XML
 HTTP/1.1 200 OK
-Content-Length: REGEX((842|844))
+Content-Length: 91
 Content-Type: application/xml
 Date: REGEX(.*)
 
 <?xml version="1.0"?>
 <orion>
-  <versionRequests>REGEX((0|1))</versionRequests>
-  <statisticsRequests>1</statisticsRequests>
-  <requestSemaphoreWaitingTime>REGEX(\d+.\d+)</requestSemaphoreWaitingTime>
-  <dbConnectionPoolWaitingTime>REGEX(\d+.\d+)</dbConnectionPoolWaitingTime>
-  <transactionSemaphoreWaitingTime>REGEX(\d+.\d+)</transactionSemaphoreWaitingTime>
-  <subCacheSemaphoreWaitingTime>REGEX(\d+.\d+)</subCacheSemaphoreWaitingTime>
-  <curlContextMutexWaitingTime>REGEX(\d+.\d+)</curlContextMutexWaitingTime>
-  <timeStatSemaphoreWaitingTime>REGEX(\d+.\d+)</timeStatSemaphoreWaitingTime>
-  <uptime_in_secs>REGEX(1?\d)</uptime_in_secs>
-  <measuring_interval_in_secs>REGEX(1?\d)</measuring_interval_in_secs>
-  <subCacheRefreshs>REGEX((0|1))</subCacheRefreshs>
-  <subCacheInserts>0</subCacheInserts>
-  <subCacheRemoves>0</subCacheRemoves>
-  <subCacheUpdates>0</subCacheUpdates>
-  <subCacheItems>0</subCacheItems>
+  <message>XML not supported in statistics operations, use JSON</message>
 </orion>
 
 
 +++++ 2. statistics in JSON
 HTTP/1.1 200 OK
-Content-Length: REGEX((693|695))
+Content-Length: REGEX(\d+)
 Content-Type: application/json
 Date: REGEX(.*)
 
 {
-    "curlContextMutexWaitingTime": "REGEX(\d+.\d+)",
-    "dbConnectionPoolWaitingTime": "REGEX(\d+.\d+)",
-    "measuring_interval_in_secs": "REGEX(1?\d)",
-    "requestSemaphoreWaitingTime": "REGEX(\d+.\d+)",
-    "statisticsRequests": "2",
-    "subCacheInserts": "0",
-    "subCacheItems": "0",
-    "subCacheRefreshs": "REGEX((0|1))",
-    "subCacheRemoves": "0",
-    "subCacheSemaphoreWaitingTime": "REGEX(\d+.\d+)",
-    "subCacheUpdates": "0",
-    "timeStatSemaphoreWaitingTime": "REGEX(\d+.\d+)",
+    "measuring_interval_in_secs": REGEX(1?\d),
+    "semWait": {
+        "curlContext": REGEX(.*.A*),
+        "dbConnectionPool": REGEX(.*.A*),
+        "request": REGEX(.*.A*),
+        "subCache": REGEX(.*.A*),
+        "timeStat": REGEX(.*.A*),
+        "transaction": REGEX(.*.A*)
+    },
     "timing": {
         "accumulated": {
-            "request": "REGEX(\d+.\d+)"
+            "total": REGEX(.*.A*)
```

Comment:
At the end it more complex that I thougth (see http://stackoverflow.com/questions/33934548/how-to-avoid-real-numbers-conversions-in-mjson-tool). I will create an issue about this to not block the PR.